tests/packages/user-args/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+build-backend = 'mesonpy'
+requires = ['meson-python']
+
+[tool.mesonpy.args]
```

Comment:
Does this need to be `mesonpy`? I think we are settling to `meson-python` as project name and it is nice to try to be consistent.

Reply:
Whops, I missed this comment :persevere: